clean-language/README-copy.md

```diff
@@ -0,0 +1,146 @@
+MASTER VERSION
+
+# Title
+> “If you want to change the world, you have to change the metaphor.”
+> - Joseph Campbell
+
+Complexity of our world is growing.
+It means that there is no single person who knows the answers to the problems we are dealing with.
+Dealing with complex problems requires other approach, it requires collaboration - people with diverse perspectives working together.
+The heart of collaboration is communication.
+Having effective communication techniques in place becomes the necessity for our success.
+Link these two sentences.
+We must invest in communication skills if we want to prepare for the future and maintain the competitive edge.
+There are many tools one can learn to improve his communication skills.
+Today we would like to talk about one of them -  Clean Language.
+
+@@ Bridge the quote with the dialog.
+(Q) for you to have a great Xebia Knowledge Exchange, it will be like what
+(A) it will be like a radar system of an old skool watch
+(Q) is there more about the radar system of the old skool watch
+(A) They are strong radars that have the power to bring forth other radars
+(Q) and whereabouts is that strong radars that have the power to bring forth other radars
+(A) its in my body
+(Q) is there more about the radar system of the old skool watch?
+(A) it is moving
+(Q) and the moving radar system, it is like what?
+(A) one big energy field
+(Q) what kind of energy field
+(A) an energy field that grows
+(Q) where is that energy field
+(A) all around me (and growing)
+(Q) then what happens
+(A) I move that energy towards others
+(Q) ...
+@@ Bridge with the next chapter.
+
+## The power of metaphor
+### Metaphors improve comprehension
+What is a metaphor? It is a figure of speech that describes an object or action in a way that isn’t literally true, but helps explain an idea or make a comparison.
+The best ones link something which is less understood to something which is more understood.
+As a result, good metaphors improve our comprehension of the ideas we might have not understood otherwise.
+Metaphors add colours to the story and make it "sticky".
+They enable us to show instead of tell.
+
+### Changing metaphors changes the way we think
+Did you know that on average we use six metaphors a minute? Surprised? Me too! They are so embedded into our speech that mostly we are unconscious about them.
+Research suggests that metaphors actually change the way we think of a concept on an unconscious level.
+In 2011 there was a study conducted in which two groups of readers have been presented with two articles describing the same crime.
+In the first article the crime was compared to a beast and in another to a disease.
+When asked to solve the crime issue, readers of the first article suggested control strategies while the second group came up with diagnostic or treatment strategies.
+So actually changing the metaphor changes how we think about the issue or an idea at hand.
```

Comment:
I like this section, it shows that metaphors are a really powerful story framing device.
I want to somehow highlight this. I'm worried people might miss it because it's in the middle.